Update syntax for docusaurus (#11)
Topl/Knowledge-Base: docs/Developer/BaaS Portal Onboarding/adding-polys.md

---
As discussed previously, polys are required in order to submit transactions to the Topl blockchain.

:::caution

You must have an address in order to add polys to your address. You can follow our documentation to generate an address using one of our Brambl libraries (see the Topl Blockchain Address page)

:::

## How can I get Polys?

It's actually pretty easy, follow these steps:

## First Step

After logging into the BaaS Portal, go to Settings and find the Purchase Polys section on the page. From there, you can transfer funds to the address directly through the UI!

![api console](https://files.readme.io/3a0e7a6-Topl_-_API_Console.gif)

You will need to wait until the funds are confirmed to use them. The confirmation status will look something like this:

![facets](https://files.readme.io/885bf20-Screen_Shot_2021-07-06_at_12.55.27_PM.png)

:::info

> Polys used on our testing network our free, however, you will have to pay to acquire polys on our main network. In addition, you can only request one faucet transaction on the testnet for every 6 hours. This ensures that there will be enough test polys for all of our users.

Reviewer: ```suggestion
Polys used on our testing network are free, however, you will have to pay to acquire polys on our main network. Additionally, you can only request one faucet transaction on the testnet every 6 hours. This ensures that there are enough test polys for all of our users.
```

:::

> If you are wanting to add polys to an account on our mainnet (toplnet), please follow the steps above but use the mainnet dropdown instead of the valhalla dropdown. You should be redirected to stripe where you can purchase polys on the main network.

Reviewer: ```suggestion
If you want to add polys to an account on our mainnet (toplnet), please follow the steps above but use the mainnet dropdown instead of the valhalla dropdown. You should be redirected to Stripe, where you can purchase polys on the main network.
```
